src/styles/blocks.scss

```diff
@@ -0,0 +1,126 @@
+* {
+  box-sizing: border-box;
+}
+
+.card {
```

Comment:
please create separate file for each BEM block - [checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md) point 11